src/components/FiltersContainer/index.js

```diff
@@ -1,33 +1,67 @@
-import { Divider, Row, Select, Typography } from "antd";
+import { Col, DatePicker, Divider, Row, Select, Typography } from "antd";
+import moment from "moment";
 import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import {
+  setSelectedFilter,
+  filtredFacts,
+  filtredByMonth,
+} from "../../redux/actions";
 
 const { Title } = Typography;
 const { Option } = Select;
 
 const FiltersContainer = () => {
+  const state = useSelector((state) => state);
+  const { filters, time } = state;
+  const dispatch = useDispatch();
+
+  const handleChangeSelect = (type, option) => {
+    const data = { name: option.children, value: option.value };
+    dispatch(setSelectedFilter(type, data));
+    dispatch(filtredFacts());
+  };
+
+  const getDisabledDate = (current) => {
+    return current && current > moment().endOf("month");
+  };
+
   return (
     <React.Fragment>
       <Divider />
-      <Title level={4} type="secondary" className="filter-title">
-        Filters:
-      </Title>
       <Row className="filters" justify="space-between">
-        <Select placeholder="Facts source">
-          <Option>1</Option>
-        </Select>
-
-        <Select placeholder="Fact verify">
-          <Option>1</Option>
-        </Select>
-
-        <Select placeholder="Animal type">
-          <Option>1</Option>
-        </Select>
-
-        <Select placeholder="Facts swear words">
-          <Option>Yes</Option>
-          <Option>No</Option>
-        </Select>
+        {filters.map((filter, i) => (
+          <Col key={String(i)}>
+            <Title level={5} type="secondary">
+              {filter.title}
+            </Title>
+            <Select
+              placeholder={filter.title}
+              value={filter.value.value}
+              onChange={(_, option) => {
+                handleChangeSelect(filter.type, option);
```

Comment:
You can create a function wrapper to simplify this callback usage. Like:
cosnt handleChangeSelect = useCallback((filterType) => (_, option) => {
    // function body
}, []);

And then use it like this:
onChange(handleChangeSelect(filter.type));